sys/mips/include/cheriasm.h

```diff
@@ -96,7 +96,7 @@
  * This will be removed once all live bitfiles and toolchain have been
  * updated.
  */
-#if (defined(CPU_CHERI) && !defined(CPU_CHERI128))
+#if 0
```

Comment:
This workaround should be long since unnecessary; `cmove` should work everywhere.

Reply:
Yes, I meant to do that as a followup change.